packages/cosmic-proto/README.md

```diff
@@ -1,9 +1,232 @@
-# Protobuf Stubs for Cosmic Swingset
+# @agoric/cosmic-proto
 
-This package exports JavaScript and TypeScript generated from Cosmic Swingset
-Protobuf IDL so that clients can communicate with the Agoric chain.
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/545047/188804067-28e67e5e-0214-4449-ab04-2e0c564a6885.svg" width="80"><br />
+    Protobuf interfaces for Agoric on Cosmos
+</p>
 
-Package exports are defined in `exports` and also made available in the
-`swingset/` file path for backwards compatibility.
 
-See also [MAINTAINERS.md](MAINTAINERS.md).
+## install
+
+```sh
+npm install @agoric/cosmic-proto
+```
+## Table of contents
+
+- [Usage](#usage)
+    - [RPC Clients](#rpc-clients)
+    - [Composing Messages](#composing-messages)
+        - Cosmos, CosmWasm, and IBC
```

Comment:
For future reference, please search the readme for all instances of (case-insensitive) "cosmwasm" and remove them as they may confuse somebody writing an Agoric smart contract.